src/Blazored.Toast/Blazored.Toast.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageTags>Blazored Blazor Razor Components Toast Popup Notification</PackageTags>
     <Description>A JavaScript free Toast library for Blazor and Razor Components applications.</Description>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>2.0.9</Version>
+    <Version>2.0.10-Preview2</Version>

```

Comment:
Could you back this change out, we'll sort version numbers when we release.

Reply:
Done - Sorry that slipped in -- I publish to a local directory to use the project and I need the larger version number to drive an update.